feature/settings/import/src/main/kotlin/app/k9mail/feature/settings/import/ui/SettingsImportScreen.kt

```diff
@@ -31,6 +31,7 @@ fun SettingsImportScreen(
                     ButtonIcon(
                         onClick = onBack,
                         imageVector = Icons.Outlined.ArrowBack,
+                        contentDescription = stringResource(R.string.settings_import_back_button),
```

Comment:
For consistency we should use the string that is (automatically) used in all other screens.
```suggestion
                        contentDescription = stringResource(androidx.appcompat.R.string.abc_action_bar_up_description),
```

Reply:
Ah nice, I was looking for this, I couldn't imagine there isn't already a string for it.